### PR TITLE
refactor: Stop humanizing properties and attributes in autocomplete

### PR DESCRIPTION
**Pull request by kof**
## Description

When user types a dash separated attribute in properties, it doesn't feel natural when autocomplete shows humanized version of it

## Steps for reproduction

1. open settings
2. add new property, start typing
3. see properties listed either camel case (properties) or kebab case (attributes)

## Code Review

- [ ] hi @TrySound , I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Comment by TrySound** [comment]
autocomplete names do not match with property names

image
image

File: apps/builder/app/builder/features/settings-panel/controls/url.tsx [hunk 1]
```diff
@@ -476,7 +477,7 @@ export const UrlControl = ({
 
   const BaseControl = modes[mode].control;
 
-  const label = getLabel(meta, propName);
+  const label = humanizeString(getLabel(meta, propName));
```
Comment (TrySound) [inline apps/builder/app/builder/features/settings-panel/controls/url.tsx:480]: getLabel seem not needed anymore `humanizeString(meta.label ?? propName)`

Comment (kof) [reply apps/builder/app/builder/features/settings-panel/controls/url.tsx:480]: done

**Comment by kof** [comment]
> autocomplete names do not match with property names

part of a change in the other PR because there are more changes that depend on it

**Pull request merged**